### DISCUSSION TITLE
`NNN_FCOLORS` -> `LS_COLORS`

### DISCUSSION THREAD

**Discussion by ghost** [jarun/nnn#1623] [Ideas]
The `NNN_FCOLORS` variable defines yet another colorscheme format. Since `${LS_COLORS}` is provided for the same purpose, could the variable (or the format, at least) be integrated into NNN? [Related: matching `ls` colors](https://github.com/jarun/nnn/issues/1463)

**Comment by jarun** [comment]
Sorry, no plans.

With the highly optimal hashing algorithm implemented in `nnn` today to resolve colors of extensions, I don't see any merit in even a single extra condition check for runtime resolution.
